addons/udes_stock/models/stock_quant_package.py

```diff
@@ -88,7 +88,11 @@ def find_move_lines(self, aux_domain=None):
 
         if aux_domain is None:
             aux_domain = [("state", "not in", ["done", "cancel"])]
-        domain = [("package_id", "in", self.ids)] + aux_domain
+        domain = aux_domain + [
+            "|",
+            ("result_package_id", "in", self.ids),
```

Comment:
Q: Is `result_package_id` a stock.quant.package field? I can only see it being defined as a move_line field in the codebase.

Reply:
It is a move_line field as we are searching for move lines that use this package which have multiple links to the package model hence the 2 fields.

Reply:
oh yeah, thought it was searching packages for some reason